frontend/src/features/account/Login.scss

```diff
@@ -1,44 +1,49 @@
 @import '../../styles.scss';
 @import '../../colors.scss';
-
 .unauth {
   font-size: 24px;
   margin: 91px auto;
-
   .row {
     justify-content: center;
   }
-
   .block {
     background-color: #f2f2f2;
     padding: 43px 0;
     max-width: 776px;
+    text-align: center;
   }
-
   h1 {
     padding-bottom: 18px;
     font-family: 'BCSans', Fallback, sans-serif;
     font-weight: 700;
     font-size: 24px;
     color: $text-color;
   }
-
+  h6 {
+    margin-bottom: 20px;
+  }
   p {
-    font-size: 16px;
+    font-size: 12px;
     color: $text-color;
     font-family: 'BCSans', Fallback, sans-serif;
     line-height: 130%;
-    width: 75%;
+    margin-left: 30px;
+    margin-right: 30px;
+    margin-bottom: -10px;
```

Comment:
What's going on here that we need a negative margin?

Reply:
fixed in next pr about to put up